program/Cargo.toml

```
@@ -18,21 +18,15 @@ borsh = { version = "1.5.1", features = ["derive", "unstable__schema"] }
 num-derive = "0.4"
 num-traits = "0.2"
 num_enum = "0.7.3"
-solana-program = { git = "https://github.com/anza-xyz/agave.git", version = "=2.1.0" }
+solana-program = "2.1"
 thiserror = "1.0.63"
 
 [dev-dependencies]
-solana-program-test = { git = "https://github.com/anza-xyz/agave.git", version = "=2.1.0" }
-solana-sdk = { git = "https://github.com/anza-xyz/agave.git", version = "=2.1.0" }
-solana-vote-program = { git = "https://github.com/anza-xyz/agave.git", version = "=2.1.0" }
+solana-program-test = "2.1"
+solana-sdk = "=2.1"
```

This probably shouldn't pin just one crate:
```suggestion
solana-sdk = "2.1"
```

---

Yep, that was a typo.